src/HTTP/HTTPContext.php

```diff
@@ -45,14 +45,18 @@ class HTTPContext
 		 * @return string|null
 		 */
 		public static function getClientIP() {
-			if (!isset($_SERVER))
+			static $ip;
```

Comment:
Keeping in consistency with the rest of the project, `$ip` should be defined as a static on the `HTTPContext` class and referred to as `self::$ip`. Unless there is some kind of low-level optimization behind this?

Reply:
Fixed, I doubt there is a major gain by doing this, other than a property access call saved but hey they are not so expensive